packages/core/tocco-ui/src/Range/Range.js

```diff
@@ -19,7 +19,7 @@ import {
  * Allows to render EditableValues as a range. The value can be switched between a range or single value.
  */
 const Range = props => {
-  const {value, events, readOnly, type, options} = props
+  const {value, events, readOnly, type, options, fromText, toText} = props
   const hasRangeValue = typeof value === 'object' && value && value.isRangeValue
```

Comment:
While you are at it :)
Can we shorten this? 
`const hasRangeValue = typeof value === 'object' && value?.isRangeValue`